datafusion/physical-expr/src/window/aggregate.rs

```diff
@@ -80,6 +80,14 @@ impl WindowExpr for PlainAggregateWindowExpr {
     }
 
     fn field(&self) -> Result<Field> {
+        // TODO: Fix window function to always return non-null for count
```

Comment:
I don't understand this comment -- can we please file a ticket to track it (and add the ticket reference to the comments)?